### PR TITLE
CAM-13376: docs(content): Added Camunda Community Hub Information 

### PR DESCRIPTION
**Pull request by celanthe**
- Added link to the Community Hub to the extensions page
- Added link to the Camunda Community Hub community repository
- Added additional context regarding the Camunda Community Hub

Related to #915

### CODE REVIEW

File: content/introduction/extensions.md [hunk 1]
```diff
@@ -16,12 +16,14 @@ Camunda Platform is developed by Camunda as an open source project in collaborat
...
 # Community Extensions
 
-Camunda supports the community in its effort to build additional community extensions under the Camunda Platform umbrella. Such community extensions are maintained by the community and are not part of the commercial Camunda Platform product.
+Camunda supports the community in its effort to build additional community extensions under the Camunda Platform umbrella. Such community extensions are maintained by the community and are not part of the commercial Camunda Platform product. 
 
 {{< note title="Camunda Support" class="warning" >}}
   Camunda does not support community extensions as part of its commercial services to enterprise subscription customers
 {{< /note >}}
 
+The [Camunda Community Hub](https://github.com/camunda-community-hub) is a GitHub Organization that serves as the home of Camunda open source community-contributed extensions. You can migrate  extension, search for existing ones, or help maintainers with contributions.
```
Comment (akeller) [inline content/introduction/extensions.md:25]: > You can migrate  extension, search for existing ones, or help maintainers with contributions.

This sentence doesn't quite make sense. Can you take another look at this @celanthe?

File: content/introduction/extensions.md [hunk 2]
```diff
@@ -55,8 +57,11 @@ The following is a list of current (unsupported) community extensions:
 
 ## Building a Community Extension
 
-Do you have a great idea around open source BPM you want to share with the world? Awesome! Camunda will support you in building your own community extension. Have a look at our [contribution guidelines](http://camunda.org/community/contribute.html) to find out how to propose a community project.
+Do you have a great idea around open source BPM you want to share with the world? Awesome! Camunda will support you in building your own community extension. Have a look at our [contribution guidelines](http://camunda.org/community/contribute.html) to find out how to propose a community project, or contribute code to the core Camunda codebase. 
+
+You can also visit the Camunda Community Hub's [community repository](https://github.com/camunda-community-hub/community) to learn more about migrating your community extension into the community hub, benefits to joining the Camunda Community Hub Organization, contributing to an extension, and much more.
```
Comment (akeller) [inline content/introduction/extensions.md:62]:
```suggestion
You can also visit the Camunda Community Hub [community repository](https://github.com/camunda-community-hub/community) to learn more about migrating your community extension into the community hub, benefits to joining the Camunda Community Hub Organization, contributing to an extension, and much more.
```

**Review by akeller — CHANGES_REQUESTED** [review]
Just a couple small things to fix.

**Comment by akeller** [comment]
I just realized this was in the camunda-docs-manual repo, meaning this content lives with the versioned content. We'll need to make sure this persists across all new versions. Shouldn't be a problem, but I wanted to make sure it was known.

**Comment by yanavasileva** [comment]
I have created CAM-13376 to track the goal and progress of the contribution. Backporting to previous versions will be take care of Runtime team.

**Review by ThorbenLindhauer — CHANGES_REQUESTED** [review]
The changes look good to me. As with the other PR, I propose we remove the list of extensions from the docs at this point (https://github.com/camunda/camunda-docs-manual/blob/df7051477e0e4872c0d4aa62f98c88bdb8b679fe/content/introduction/extensions.md#list-of-community-extensions) and point people to the hub instead. What do you think?

**Review by celanthe — COMMENTED** [review]
Made changes requested and submitted a new commit!

**Review by ThorbenLindhauer — APPROVED** [review]

**Comment by ThorbenLindhauer** [comment]
Hey @celanthe,

Thanks for your contribution, I merged it together with #915 via d1f2fcf.

Cheers,
Thorben

**Pull request closed without merging**